stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml

```diff
@@ -88,6 +88,9 @@ spec:
           echo "Copying bootstrap configs";
           cp -Lrf /bootstrap/* /artifactory_extra_conf/;
           {{- end }}
+          mkdir -p {{ .Values.artifactory.persistence.mountPath }}/access/etc;
+          cp -Lrf /tmp/bootstrap.creds {{ .Values.artifactory.persistence.mountPath }}/access/etc/bootstrap.creds;
```

Comment:
Take it from `/tmp/access/bootstrap.creds`?

Reply:
Done